autopilot/walletmaintainer/wallet.go

```diff
@@ -79,18 +79,24 @@ func (w *walletMaintainer) PerformWalletMaintenance(ctx context.Context, cfg api
 		}
 	}
 
-	// check whether the wallet needs to be redistributed
-	wantedNumOutputs := 10
+	// calculate num outputs
+	const maxOutputs = 100
 	amount := contractor.InitialContractFunding.Mul64(10)
-	if balance.Cmp(amount.Mul64(uint64(wantedNumOutputs))) < 0 {
+	numOutputs := balance.Div(amount).Big().Uint64()
+	if numOutputs > maxOutputs {
+		numOutputs = maxOutputs
+	}
+
+	// skip maintenance if wallet balance is too low
+	if numOutputs < 10 {
```

Comment:
(nit) do you want to update the warning now that we check for outputs instead of balance?

Reply:
I think it's fine?

Reply:
The condition is `numOutputs < 10` but the warning mentions the balance being too low with the current balance rather than the fact that `< 10` outputs triggered the warning

Reply:
I mean `< 10` didn't really trigger the warning. Why would we want to specify "10" as a meaningful number. We want to split into 100, lower is fine, below 10 we don't bother. This warning essentially says: your wallet is virtually empty, we won't even bother redistributing it. We're talking sub 1KS wallets here. I don't mind updating the warning but I feel we would be needless specifying a very arbitrary condition that doesn't mean anything.